main.go

```diff
@@ -53,6 +56,8 @@ func init() {
 	kingpin.Flag("labels", "A set of labels to restrict the list of affected pods. Defaults to everything.").StringVar(&labelString)
 	kingpin.Flag("annotations", "A set of annotations to restrict the list of affected pods. Defaults to everything.").StringVar(&annString)
 	kingpin.Flag("namespaces", "A set of namespaces to restrict the list of affected pods. Defaults to everything.").StringVar(&nsString)
+	kingpin.Flag("include-pattern", "Regex pattern for pod names to include.  All included by default").StringVar(&includePattern)
```

Comment:
How about calling the flags `included-pod-names` and `excluded-pod-names` to match the naming style of the other flags.